src/cli.ts

```diff
@@ -1,36 +1,79 @@
 #!/usr/bin/env node
-import mri from "mri";
-import { mkdist } from "./index";
+import { defineCommand, runMain } from 'citty';
+import { version } from '../package.json';
+import { mkdist, MkdistOptions } from "./index";
 
-async function main() {
-  const arguments_ = mri(process.argv.splice(2));
+const main = defineCommand({
+  meta: {
+    version,
+    name: 'mkdist',
+    description: 'Lightweight file-to-file transformer',
+  },
+  args: {
+    rootDir: {
+      type: 'positional',
+      description: 'Project root directory',
+      required: false,
+      default: '.',
+    },
+    src: {
```

Comment:
Would you please also add `--dir` argument? This will be an alternative to positional `rootDir` and to be preferred in the future for the CLIs.

Example:
- https://github.com/unjs/nitro/blob/main/src/cli/common.ts
- https://github.com/unjs/nitro/blob/main/src/cli/commands/dev.ts#L21